### ISSUE TITLE
Where is the modelutils module? 

### ISSUE THREAD

**Issue by BadisG** [oobabooga/text-generation-webui#1444]
### Describe the bug

Hello,

I can't use the triton models anymore because it says it doesn't have the modelutils module.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

python server.py --monkey-patch --model_type llama

### Logs

```shell
Traceback (most recent call last):
File “/mnt/d/Large Language Models/text-generation-webui-TRITON/server.py”, line 100, in load_model_wrapper
shared.model, shared.tokenizer = load_model(shared.model_name)
File “/mnt/d/Large Language Models/text-generation-webui-TRITON/modules/models.py”, line 118, in load_model
from modules.monkey_patch_gptq_lora import load_model_llama
File “/mnt/d/Large Language Models/text-generation-webui-TRITON/modules/monkey_patch_gptq_lora.py”, line 15, in
from modules.GPTQ_loader import find_quantized_model_file
File “/mnt/d/Large Language Models/text-generation-webui-TRITON/modules/GPTQ_loader.py”, line 15, in
from modelutils import find_layers
ModuleNotFoundError: No module named ‘modelutils’
```

### System Info

```shell
Windows 10 + WSL
Main repo of ooba's webui + Triton branch of GPTQ-for-LLaMa + Main repo of alpaca_lora_4bit
```

**Comment by LaaZa** [comment]
will be fixed by #1408

**Comment by BadisG** [comment]
@LaaZa Oh ok, thanks for the info!
